cleanlab/filter.py

```diff
@@ -236,50 +235,42 @@ def multiclass_crossval_predict(pyx, labels):
     return pred
 
 
-def get_noise_indices(
-        s,
-        psx,
-        inverse_noise_matrix=None,
-        confident_joint=None,
-        frac_noise=1.0,
-        num_to_remove_per_class=None,
-        prune_method='prune_by_noise_rate',
-        sorted_index_method=None,
-        multi_label=False,
-        n_jobs=None,
-        verbose=0,
-):
-    """Returns the indices of most likely (confident) label errors in s. The
+def find_label_issues(labels, psx, confident_joint=None, frac_noise=1.0,
+                      num_to_remove_per_class=None, filter_by='prune_by_noise_rate',
+                      return_indices_ranked_by=None, multi_label=False, n_jobs=None, verbose=0):
+    """By default, this method returns a boolean mask for the entire dataset where True represents
+    a label issue and False represents an example that is confidently/accurately labeled.
+
+    You can return ONLY the indices of the label issues in your dataset, by setting
+    return_indices_ranked_by = {`prob_given_label`, `normalized_margin`}.
+
     number of indices returned is specified by frac_of_noise. When
     frac_of_noise = 1.0, all "confident" estimated noise indices are returned.
```

Comment:
is there rationale for not allowing `frac_of_noise` to be > 1 (i.e. to get higher recall in label-error detection)?

Reply:
FYI the current code seems to still run with `frac_noise` > 1 even though it has no effect, should probably raise error.

Reply:
It's an easter egg use case. Basically, I want the ability to do your first comment without confusing users so i say it needs to be between 0 and 1, but for advanced users, you should be able to increase it... but eventualy... things will break. Instead of handling all the edge cases, I just tell the users what the input should be and if they want to break it they can.

Perhaps I could add a warning saying a succinct version of the above (i.e., `Warning `frac_noise` is intended to be between 0 and 1. You can use large values, but things may break. Use at your own risk``)

Thoughts?

Reply:
I like raising that warning when `frac_noise` is detected to be > 1